akka-actor-typed/src/main/scala/akka/actor/typed/internal/Supervision.scala

```diff
@@ -81,10 +72,15 @@ private abstract class AbstractSupervisor[I, Thr <: Throwable](strategy: Supervi
       val unwrapped = UnstashException.unwrap(t)
       strategy.logLevel match {
         case Logging.ErrorLevel =>
-          ctx.asScala.log.error(unwrapped, "Supervisor {} saw failure: {}", this, unwrapped.getMessage)
+          ctx.asScala.log.error("Supervisor {} saw failure: {}", Array(this, unwrapped.getMessage))
         case Logging.WarningLevel =>
-          ctx.asScala.log.warning(unwrapped, "Supervisor {} saw failure: {}", this, unwrapped.getMessage)
-        case level => ctx.asScala.log.log(level, "Supervisor {} saw failure: {}", this, unwrapped.getMessage)
+          ctx.asScala.log.warn("Supervisor {} saw failure: {}", Array(this, unwrapped.getMessage))
+        case Logging.InfoLevel =>
+          ctx.asScala.log.info("Supervisor {} saw failure: {}", Array(this, unwrapped.getMessage))
+        case Logging.DebugLevel =>
+          ctx.asScala.log.debug("Supervisor {} saw failure: {}", Array(this, unwrapped.getMessage))
+        //TODO check this debug case is actually best option when other level is found
+        case _ => ctx.asScala.log.debug("Supervisor {} saw failure: {}", Array(this, unwrapped.getMessage))
```

Comment:
Same here, use `org.slf4j.event.Level` and there will be no unknowns because it is an enum

Reply:
yep